webhooks-extension/src/components/WebhookDisplayTable/WebhookDisplayTable.js

```diff
@@ -30,22 +32,79 @@ const {
   TableHeader,
 } = DataTable;
 
-export class WebhookDisplayTable extends Component {
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
```

Comment:
I'm not best placed to comment here given my level of knowledge in this area ..... but is there no way all this styling could be done in the scss rather than in the js?  Can you not provide classNames in all the relevant places? Maybe @ncskier can comment better.

Reply:
I've moved this into its own jsx file that I'll import